### DISCUSSION TITLE
Icon in Mac

### DISCUSSION THREAD

**Discussion by singuerinc** [hoppscotch/hoppscotch#1876] [General]
Hi, is there anything we can do to improve the icon, following the guidelines from Apple?
This is how the icon looks today:

<img width="598" alt="Screenshot 2021-10-14 at 13 04 22" src="https://user-images.githubusercontent.com/503676/137305931-c50d23dd-cbd9-44f9-9847-a6fa7c60bd17.png">

Thanks!

**Comment by liyasthomas** [comment]
Hi @singuerinc, let me know whether you installed the latest Hoppscotch build or not. We've updated the app icons to match the latest guidelines on our recent v2.x versions, here are some screenshots I just took from my end.

MacBook Air - macOS Big Sur
Google Chrome - Version 94.0.4606.81 (Official Build) (arm64)

<img width="1440" alt="Screenshot 2021-10-14 at 5 54 51 PM" src="https://user-images.githubusercontent.com/10395817/137316960-ae11faaa-1c5e-45c6-bc91-c73a2ba8bb34.png">

<img width="1440" alt="Screenshot 2021-10-14 at 5 57 31 PM" src="https://user-images.githubusercontent.com/10395817/137317050-e6fbf1b8-b829-4edb-8789-d4c2c5d825e2.png">

Even though the app serves the latest build, there are chances it might be using the cached old icon assets. Please uninstall the pervious PWA and install the latest one from https://hoppscotch.io.

Let me know if that resolves this issue.

**Reply by singuerinc** [reply]
Re-installing solved the issue, I guess I updated from an old version, and it kept the old icon cached somewhere. Thanks!

**Reply by AndrewBastin** [reply]
I think its a Chrome issue, this is the same across operating systems and is kinda annoying! (Don't know if other PWA capable browsers check this as well) ¯\_(ツ)_/¯
